templates/osbs-buildroot-imagestream.yml.j2

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ImageStream
+metadata:
+  annotations:
+    openshift.io/image.insecureRepository: "true"
```

Comment:
This should be parametrized as well - Fedora's repository is secure and some day ours will be too

Reply:
ok